cmd/limactl/main.go

```diff
@@ -119,3 +119,18 @@ func handleExitCoder(err error) {
 		return
 	}
 }
+
+func WrapArgsError(argFn cobra.PositionalArgs) cobra.PositionalArgs {
```

Comment:
Please consider adding a comment line

Reply:
Thanks for quick review, updated in 48ae695f7bda507f9209d2226c31c72959adb0cd.